plugins/azure/resoto_plugin_azure/resource/compute.py

```diff
@@ -2447,7 +2478,7 @@ class AzureVirtualMachineIdentity:
 
 
 @define(eq=False, slots=False)
-class AzureVirtualMachine(AzureResource):
+class AzureVirtualMachine(AzureResource, BaseInstance):
```

Comment:
I do not see the instance_status field mapped.
It should be derived from the provisioning_state value.